src/views/consumer/view.ejs

```diff
@@ -1,17 +1,47 @@
-<%- include("partials/header"); %>
-<div class="form-background">
+<% if (locals?.isDeveloper) { %>
+    <%- include("../partials/header"); %>
+<% } else { %>
+    <%- include("partials/header"); %>
+<% } %>
```

Comment:
Not a massive fan of having these types of conditions in the consumer templates... 

Previously we had a consumer template that just included partials that were shared between publisher/consumer side, that way there's no chance of this stuff ever being accidentally exposed. Is there a reason we can't still do it that way?

Reply:
Yes but there's the strong call from UX to use the same view for all data views... Maintaining two near identical views makes very little sense from a development point of view.

Reply:
I think that was more about the data /metadata views looking identical. We could just share those and keep the dev-specific stuff in the dev template.

Reply:
Ok lets merge it for now and revisit.